packages/shared/src/components/modals/NewCommentModal.tsx

```diff
@@ -173,14 +173,15 @@ export default function NewCommentModal({
       contentRef={modalRef}
       onRequestClose={confirmClose}
       kind={Modal.Kind.FlexibleCenter}
-      size={Modal.Size.Small}
+      size={Modal.Size.Medium}
       tabs={Object.values(CommentTabs)}
       {...props}
+      className="min-h-[25rem]"
```

Comment:
I'm not really sure about this one, should it not rather be something the guidelines handle instead of customizing a modal just for this?

Reply:
I agree, why do we set a custom size here? You have already changed to `Modal.Size.Medium` if there is a height issue with that `Modal.Size.Medium` it should be handled in the modal file. Is there a comment modal-specific height requirement?

Reply:
@el yes, there is a height requirement due to the UX. So when there is not much comment to preview, the modal wouldn't shrink. I was just worried that the other Modals using `Medium` might not have a height requirement.

Reply:
Can you please check this with design? If the height requirement is comment specific, we can leave it like this, if it applies to all the modals with the size & kind, you can move the `min-height` to `modal.tsx`.

Reply:
After discussing with Tomer. The height requirement is only specific to this, but having the `640px` height will be then a requirement when the modal is a `FixedCenter - Medium` but since this is a `FlexibleCenter - Medium` the said requirement does not apply here.

We should be good to go with the changes here. Thanks for taking a look at this 🚢

Reply:
After Tomer's recent feedback. I will implement a new kind of modal. Kindly wait for it to get implemented before reviewing. Thanks!